.pre-commit-config.yaml

```diff
@@ -49,15 +15,8 @@ repos:
       - id: check-yaml
       - id: debug-statements
       - id: check-added-large-files
-        args: ['--enforce-all','--maxkb=1054']
+        args: ["--enforce-all", "--maxkb=1054"]
       - id: end-of-file-fixer
-        exclude: ".*(.fits|.fts|.fit|.header|.txt|tca.*)$"
+        exclude: ".*(.fits|.fts|.fit|.header|.txt|tca.*|.json)$|^CITATION.rst$"
       - id: mixed-line-ending
         exclude: ".*(.fits|.fts|.fit|.header|.txt|tca.*)$"
-  - repo: https://github.com/codespell-project/codespell
-    rev: v2.2.6
-    hooks:
-      - id: codespell
-        args: ['--config setup.cfg']
-ci:
-  autofix_prs: false
```

Comment:
This didn't either? 
We should also disable updates if the plan is to roll them out via the template?

Reply:
We should add this to the template, asap.

>  We should also disable updates if the plan is to roll them out via the template?

hmm, this is a good and interesting question. Yes, we should roll out updates via the template, this would mean we go from one (pre-commit) to one (template update) auto PR rather than one to two. We should figure out how we can automate bumping the versions in the template though or they will surely get forgotten.

https://github.com/sunpy/package-template/issues/33